<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAward</class>
 <widget class="QDialog" name="DialogAward">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>644</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonNew">
        <property name="text">
         <string>Новый</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxName">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widgetTools" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="pushButtonGo">
        <property name="text">
         <string>Выгрузить данные</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Настройки</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonColorBoundText">
           <property name="text">
            <string>Цвет обводки текста</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButtonColorBound">
           <property name="text">
            <string>Цвет обводки флага</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonColorText">
           <property name="text">
            <string>Цвет текста</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonFont">
           <property name="text">
            <string>Шритф текста</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBoxWidthTextBound">
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Ширина обводки:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Гимн</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonHymnCounty">
           <property name="text">
            <string>Страна</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonHymnRegion">
           <property name="text">
            <string>Регион</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonHymnCity">
           <property name="text">
            <string>Город</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonHymnClub">
           <property name="text">
            <string>Клуб</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Флаг</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonFlagCounty">
           <property name="text">
            <string>Страна</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonFlagRegion">
           <property name="text">
            <string>Регион</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonFlagCity">
           <property name="text">
            <string>Город</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonFlagClub">
           <property name="text">
            <string>Клуб</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Картинка</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="pushButtonLoadImg">
           <property name="text">
            <string>Загрузить картинку...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ширина</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelImageWidth">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Высота</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelImageHeight">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Координаты</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="2">
             <widget class="QRadioButton" name="radioButtonFlag1">
              <property name="text">
               <string>Флаг 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="radioButtonFlag0">
              <property name="text">
               <string>Флаг 1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="radioButtonName2">
              <property name="text">
               <string>ФИО 3</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QRadioButton" name="radioButtonFlag2">
              <property name="text">
               <string>Флаг 3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="radioButtonName3">
              <property name="text">
               <string>ФИО 4</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButtonName0">
              <property name="text">
               <string>ФИО 1</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QRadioButton" name="radioButtonFlag3">
              <property name="text">
               <string>Флаг 4</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="radioButtonName1">
              <property name="text">
               <string>ФИО 2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButtonTC">
              <property name="text">
               <string>Весовая категория</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Ширина</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Высота</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxX">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100500</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxY">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100500</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBoxWidth">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>100500</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBoxHeight">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>100500</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Название</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonNameCounty">
           <property name="text">
            <string>Страна</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonNameRegion">
           <property name="text">
            <string>Регион</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonNameCity">
           <property name="text">
            <string>Город</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonNameClub">
           <property name="text">
            <string>Клуб</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="DialogAwardWidget" name="widgetAward" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialogAwardWidget</class>
   <extends>QWidget</extends>
   <header>dialogaward.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxName</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>pushButtonLoadImg</tabstop>
  <tabstop>radioButtonName0</tabstop>
  <tabstop>radioButtonName1</tabstop>
  <tabstop>radioButtonName2</tabstop>
  <tabstop>radioButtonName3</tabstop>
  <tabstop>radioButtonFlag0</tabstop>
  <tabstop>radioButtonFlag1</tabstop>
  <tabstop>radioButtonFlag2</tabstop>
  <tabstop>radioButtonFlag3</tabstop>
  <tabstop>spinBoxX</tabstop>
  <tabstop>spinBoxY</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>spinBoxHeight</tabstop>
  <tabstop>pushButtonFont</tabstop>
  <tabstop>pushButtonColorText</tabstop>
  <tabstop>pushButtonColorBoundText</tabstop>
  <tabstop>spinBoxWidthTextBound</tabstop>
  <tabstop>pushButtonColorBound</tabstop>
  <tabstop>radioButtonFlagCounty</tabstop>
  <tabstop>radioButtonFlagRegion</tabstop>
  <tabstop>radioButtonFlagCity</tabstop>
  <tabstop>radioButtonFlagClub</tabstop>
  <tabstop>radioButtonHymnCounty</tabstop>
  <tabstop>radioButtonHymnRegion</tabstop>
  <tabstop>radioButtonHymnCity</tabstop>
  <tabstop>radioButtonHymnClub</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
